enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/AndeltilkjentYtelseDto.kt

```diff
@@ -1,9 +1,14 @@
 package no.nav.familie.kontrakter.ef.iverksett
 
+import java.time.LocalDate
 import java.util.UUID
 
 class AndelTilkjentYtelseDto(
-        val periodebeløp: PeriodebeløpDto,
+        val beløp: Int,
+        val fraOgMed: LocalDate,
+        val inntekt: Int,
+        val inntektsreduksjon: Int,
+        val tilOgMed: LocalDate,
```

Comment:
Ønsker vi denne breaking change med å splitte periodebeløp? 
Trenger vi inntekt?

Reply:
Skjønte inte hvorfør vi hade periodebeløp ...

Reply:
inntekt og inntektsreduksjon trenger vi for å sende utbetalingsgrad til økonomi (hvis jag har skjønt det riktigt...[favro](https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-3242))

Reply:
Tror ikke vi trenger inntekt pga det?